db/migrate/20160308155249_add_online_application_tables.rb

```diff
@@ -0,0 +1,37 @@
+class AddOnlineApplicationTables < ActiveRecord::Migration
+  def change
+    create_table :online_failures do |t|
+      t.text :received_data, null: false
+      t.timestamps null: false
+    end
+
+    create_table :online_applications do |t|
+      t.boolean :applicant_married # marital_status_married
+      t.boolean :application_threshold_exceeded # savings_and_investment_less_than_limit
+      t.boolean :application_benefits # benefit_on_benefits
+      t.boolean :application_descendants # dependent_children
+      t.integer :application_children # dependent_children_number
+      t.integer :application_income # income_total
+      t.boolean :details_refund # fee_paid
+      t.string :details_date_fee_paid # fee_date_paid
+      t.boolean :details_probate # probate_kase
```

Comment:
I think that `case` on it's own is reserved, but it can be used as part of a word.